### ISSUE TITLE
Staging Review finding: Minimal header pages: First segment of title tag should match h1

### ISSUE THREAD

**Issue by jasonday** [department-of-veterans-affairs/va.gov-team#122406]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Design System & Forms Library
**Product:** Form 21-4140
**Feature:** Employment Questionnaire

## Findings details
**VA.gov Experience Standard - issue:** [User can’t determine where a page or screen is within VA.gov or VA Mobile App.](https://design.va.gov/about/experience-standards/#user-can-determine-where-a-page-is-within-the-vagov-website)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @jasonday (Accessibility)

### Description

On the minimal header pages, the H1 and the first segment of the title tag should be the same. I realize this diverges from our current guidance for form titles in VADS. The guidance is currently being updated, so this finding is a note to revisit the title tags when you can. This is also an accessibility issue as each page should have a distinct title.

## Recommended action

On pages using the minimal header, use the same text for the h1 and the first segment of the title tag.

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** [Title tags](https://design.va.gov/content-style-guide/title-tags)
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jeana-adhoc** [comment]
I've been in conversations with @coforma-jamie, @laurenvocke-va and @jonichan-ia about the `<title>` changes.  But I don't think anything has been decided yet. Will hold on to this ticket until I know more, and we have documentation in place to support this change across all forms (both minimal form flow and the legacy form flows)

**Comment by jeana-adhoc** [comment]
Checked in with @laurenvocke-va and there's been no progress here, however she's going to talk with Danielle and there may be an update soon.
